src/semgrep_agent/findings.py

```diff
@@ -98,6 +99,61 @@ def to_dict(self, omit: Set[str]) -> Mapping[str, Any]:
         d["commit_date"] = d["commit_date"].isoformat()
         return d
 
+    def to_gitlab(self) -> Dict[str, Any]:
+        return {
+            "id": str(uuid.uuid5(uuid.NAMESPACE_URL, str(self.path))),
+            "category": "sast",
+            # CVE is a required field from Gitlab schema.  Semgrep is CVE-unaware AFAIK
+            "cve": "",
+            "message": self.message,
+            "severity": self._to_gitlab_severity(),
+            # KB note:
+            # Semgrep is designed to be a low-FP tool by design.
+            # Does hard-coding confidence make sense here?
```

Comment:
yep, we did this for SARIF as well

although now I think we should've done `"High" if "audit" not in rule_id.split(".") else "Low"` (the audit rules are to be used with human reviewers checking all results)